src/Files.App/Dialogs/ReorderSidebarItemsDialog.xaml.cs

```diff
@@ -0,0 +1,102 @@
+using CommunityToolkit.WinUI.UI;
+using Files.App.DataModels.NavigationControlItems;
+using Files.App.Extensions;
+using Files.App.Filesystem;
+using Files.App.ServicesImplementation;
+using Files.App.ViewModels.Dialogs;
+using Files.Backend.ViewModels.Dialogs;
+using Files.Shared.Enums;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.DataTransfer;
+
+namespace Files.App.Dialogs
+{
+	public sealed partial class ReorderSidebarItemsDialog : ContentDialog, IDialog<ReorderSidebarItemsDialogViewModel>
+	{
+		public ReorderSidebarItemsDialogViewModel ViewModel
```

Comment:
Where is this ViewModel?

Reply:
It's here: `src/Files.App/ViewModels/Dialogs/ReorderSidebarItemsDialogViewModel.cs`